sdk/communication/Azure.Communication.Chat/api/Azure.Communication.Chat.netstandard2.0.cs

```diff
@@ -1,11 +1,21 @@
 namespace Azure.Communication.Chat
 {
+    public partial class AddChatParticipantsErrors
+    {
+        internal AddChatParticipantsErrors() { }
+        public System.Collections.Generic.IReadOnlyList<Azure.Communication.Chat.CommunicationError> InvalidParticipants { get { throw null; } }
+    }
+    public partial class AddChatParticipantsResult
+    {
+        internal AddChatParticipantsResult() { }
+        public Azure.Communication.Chat.AddChatParticipantsErrors Errors { get { throw null; } }
+    }
     public partial class ChatClient
     {
         protected ChatClient() { }
         public ChatClient(System.Uri endpointUrl, Azure.Communication.CommunicationTokenCredential communicationTokenCredential, Azure.Communication.Chat.ChatClientOptions? options = null) { }
-        public virtual Azure.Communication.Chat.ChatThreadClient CreateChatThread(string topic, System.Collections.Generic.IEnumerable<Azure.Communication.Chat.ChatThreadMember> members, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
-        public virtual System.Threading.Tasks.Task<Azure.Communication.Chat.ChatThreadClient> CreateChatThreadAsync(string topic, System.Collections.Generic.IEnumerable<Azure.Communication.Chat.ChatThreadMember> members, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Response<Azure.Communication.Chat.CreateChatThreadResult> CreateChatThread(string topic, System.Collections.Generic.IEnumerable<Azure.Communication.Chat.ChatParticipant> participants, string? repeatabilityRequestId = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```

Comment:
I see that we are using nullable ref-types here, let's remove those too, but maybe in a follow up PR.